py/server/deephaven/parquet.py

```diff
@@ -280,6 +288,93 @@ def write(
         raise DHError(e, "failed to write to parquet data.") from e
 
 
+def write_key_value_partitioned_table(
+        table: Union[Table, PartitionedTable],
+        destination_dir: str,
+        col_definitions: Optional[List[Column]] = None,
+        col_instructions: Optional[List[ColumnInstruction]] = None,
+        compression_codec_name: Optional[str] = None,
+        max_dictionary_keys: Optional[int] = None,
+        max_dictionary_size: Optional[int] = None,
+        target_page_size: Optional[int] = None,
+        base_name: Optional[str] = None,
+        generate_metadata_files: Optional[bool] = False,
+) -> None:
+    """ Write a table to a Parquet file in a key-value partitoned format.
```

Comment:
A user is not going to know what a "key-value partitioned format" is.  Provide some details.